kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/model/networking/IngressPlanningException.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator.model.networking;
+
+public class IngressPlanningException extends RuntimeException {
```

Comment:
Is it an `IngressPlanningException` or should it be a `NetworkPlanningException`